src/components/Themeroller/ThemeTuner/ItemValue/ColorValue/__tests__/ColorValue.spec.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { render, RenderResult, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { ThemeColorItem } from '@textkernel/oneui';
+import { ColorValue } from '../ColorValue';
+
+describe('ColorValue component', () => {
+    const item = { value: '#fffff' } as ThemeColorItem;
+    const inputValue = { value: '#00000' } as ThemeColorItem;
+    const role = 'colorselector';
+
+    let view: RenderResult;
+
+    it('should render component correctly', () => {
+        const onChangeMock = jest.fn();
+        view = render(<ColorValue item={item} onChange={onChangeMock} ariaRole={role} />);
```

Comment:
`onChangeMock` is not really used, just pass `onChange={jest.fn()}`

Reply:
:+1: